ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-stable.yaml

```diff
@@ -62,6 +62,11 @@ releases:
       architecture: amd64
       channel: candidate
       version: "4.13"
+  source:
```

Comment:
stable ci does not run any test suite, so we do not need the setting here.

Reply:
remove it, thanks